INSTALL

```diff
@@ -5,7 +5,19 @@ How to install PBS using the configure script.
 
 1. Install the prerequisite packages for building PBS.
 
-  For CentOS systems you should run the following command as root:
+  For CentOS-8 systems you should run the following command as root:
+
+    Configure and enable PowerTools repo for hwloc-devel and libedit-devel packages
+      dnf install -y dnf-plugins-core
+      dnf config-manager --set-enabled PowerTools
+
+    dnf install -y gcc make rpm-build libtool hwloc-devel \
```

Comment:
An instruction can be added above saying install required packages or tools.

Reply:
Mentioned at line 11 .